src/UsgsAstroFramePlugin.cpp

```diff
@@ -273,18 +278,55 @@ return sensor_model;
 }
 
 
-csm::Model *UsgsAstroFramePlugin::constructModelFromISD(const csm::Isd &imageSupportData,
+csm::Isd UsgsAstroFramePlugin::loadImageSupportData(const csm::Isd &imageSupportDataOriginal) const{
```

Comment:
The name for this method (and its signature) is a little bit confusing, but I can't think of anything better for it. A few lines documenting what this does and why it's needed would be helpful.